### PR TITLE
fix(docker): failed to build image on Mac M1

### PR DESCRIPTION
**Pull request by yuanbohan**
## Reference

- https://stackoverflow.com/questions/71040681/qemu-x86-64-could-not-open-lib64-ld-linux-x86-64-so-2-no-such-file-or-direc
- https://github.com/nodejs/help/issues/3239

### CODE REVIEW

**Review by sunng87 — APPROVED** [review]

**Comment by sunng87** [comment]
@ZonaHex PTAL. It seems we don't have remove image optimisations using this.

File: README.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ and run via:
...
 ```
 docker pull greptime/greptimedb-dashboard
-docker run \
+docker run --rm --name greptimedb-dashboard \
   -e GREPTIMEDB_HTTP_HOST=127.0.0.1 \
   -e GREPTIMEDB_HTTP_PORT=4000 \
   -e NGINX_PORT=8080 \
-  --network host \
+  -p 8080:8080 \
   greptime/greptimedb-dashboard:latest
```
Comment (ZonaHex) [inline README.md:32]: If we don't run db in docker, we can't access db in PC.

File: docker/greptimedb.conf [hunk 2]
```diff
@@ -1,14 +1,18 @@
 server {
   listen          ${NGINX_PORT};
-  port_in_redirect off;
   root /usr/share/nginx/html/;
+  absolute_redirect off;
 
   location /api/ {
     proxy_pass http://${GREPTIMEDB_HTTP_HOST}:${GREPTIMEDB_HTTP_PORT}/;
   }
```
Comment (ZonaHex) [inline docker/greptimedb.conf:8]: Not sure if this works.

**Review by ZonaHex — COMMENTED** [review]
We should update README if 'docker run' does not works well without db in docker.

**Pull request merged**